src/components/book-item.js

```diff
@@ -13,14 +14,14 @@ const Book = ({
   };
   return (
     <li>
-      <div>
-        <div>
-          <div>
-            <h4>{category}</h4>
-            <h2>{title}</h2>
+      <div className={classes.main}>
+        <div className={classes.content}>
+          <div className={classes.info}>
+            <h4 className={classes.fiction}>{category}</h4>
+            <h2 className={classes.title}>{title}</h2>
             <h6>{author}</h6>
-            <div>
-              <button type="button">Comments</button>
+            <div className={classes.cre}>
+              <button type="button">Comments |</button>
               <div />
               <button
                 className="button-outline"
```

Comment:
- Please make sure the Form and books font, colour, font size and design are the same as that of the Zeplin design
Zeplin Design | Your Design

![Screenshot from 2022-07-16 09-39-03](https://user-images.githubusercontent.com/44624138/179347581-e7c1e148-24aa-424f-a12b-c0d94f9cd14a.png)
![Screenshot from 2022-07-16 08-18-04](https://user-images.githubusercontent.com/44624138/179347586-cc8a011a-3322-4724-bcb3-4e24fcb598d1.png)